pennylane/plugins/tests/__init__.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This subpackage provides integration tests for the devices with PennyLane's core
+functionalities. At the moment, the tests only run on devices based on the 'qubit' model.
+
+To run the tests, navigate to the parent directory of this file and run (i.e., for 'default.qubit'):
+
+>>> python3 -m pytest tests/* --device default.qubit --shots 1234 --analytic False
+
+The command line arguments are optional.
+
+* If `--device` is not given, the tests are run on the qubit core devices that ship with PennyLane.
+
+* If `--shots` is not given, a default of 50000 is used.
```

Comment:
Does the 50000 need to be modified?

Reply:
might be worth specifying that this option does not apply if `analytic=True`

Reply:
We need a better term than 'analytic', don't we?

Reply:
Yes!! But I don't have a better one either...

Yes, I thought I changed the 50000 to 10000, but something must have merged in.